proknow-sdk/Patient/Entities/EntityScorecards.cs

```diff
@@ -73,7 +73,7 @@ public EntityScorecards(ProKnowApi proKnow, string workspaceId, string entityId)
             var customMetricIdsAndObjectives = resolvedCustomMetrics.Select(c => c.ConvertToScorecardSchema()).ToList();
 
             // Request the creation
-            var route = $"/workspaces/{_workspaceId}/patient/{_entityId}/metrics/sets";
+            var route = $"/workspaces/{_workspaceId}/patients/{_entityId}/metrics/sets";
             var requestSchema = new EntityScorecardItem(null, null, null, null, null, name, computedMetrics, customMetricIdsAndObjectives);
```

Comment:
Update all instances in this class of the phrase `entity scorecard` in the XML comments to `patient scorecard`